schemas/+acquisition/SessionTest.m

```diff
@@ -0,0 +1,202 @@
+%{
+# General information of a session
+-> acquisition.SessionStarted
+---
+session_start_time          : datetime                      # start time
+session_end_time=null       : datetime                      # end time
+(session_location) -> lab.Location
+-> task.SubtaskLevelParameterSet
+-> lab.ManipulationType
+stimulus_bank = ''          : varchar(255)                  # path to the function to generate the stimulus
+stimulus_commit = ''        : varchar(64)                   # git hash for the version of the function
+stimulus_set                : tinyint                       # an integer that describes a particular set of stimuli in a trial
+session_performance         : float                         # percentage correct on this session
+num_trials=-1               : int(11)                       # Number of trials for the session
+num_trials_try=null         : tinyblob                      # Accumulative number of trials for each try of the session
+session_narrative = ''      : varchar(512)                  # descriptive string
+session_protocol=null       : varchar(255)                  # function and parameters to generate the stimulus
+session_code_version=null   : blob                          # code version of the stimulus, usually two numbers maze_version=2.1, code_version = 4.0. In the future: a githash?
+is_bad_session=0            : boolean                       # Flag that indicates if this session had any issues
+session_comments=''         : varchar(2048)                 # Text to indicate some particularity of the session (e.g. state the issues in a bad session)
+%}
+
+classdef SessionTest < dj.Imported
```

Comment:
For this `SessionTest`, are we planning to replace the current `SessionStart` or `Session`? `lab.ManipulationType` is the new field here?

Reply:
I prefer to mantain the same Session table and replace foreign key:
-> task.TaskLevelParameterSet. (task, level and set_id)
with
-> task.SubtaskLevelParameterSet.  (task, subtask, level and set_id)
Two new fields:
task.SubtaskLevelParameterSet (subtask)
lab.ManipulationType (mainpulation_type)

Reply:
Solve with category